messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/MessagingSubsystemParser_1_0.java

```diff
@@ -457,8 +457,8 @@
                                                         ConnectionFactoryAttributes.Common.DUPS_OK_BATCH_SIZE,
                                                         ConnectionFactoryAttributes.Common.TRANSACTION_BATCH_SIZE,
                                                         ConnectionFactoryAttributes.Common.BLOCK_ON_ACKNOWLEDGE,
-                                                        ConnectionFactoryAttributes.Common.BLOCK_ON_DURABLE_SEND,
                                                         ConnectionFactoryAttributes.Common.BLOCK_ON_NON_DURABLE_SEND,
+                                                        ConnectionFactoryAttributes.Common.BLOCK_ON_DURABLE_SEND,
```

Comment:
this change is wrong. In the XSD, block-on-durable-send is before block-on-non-durable-send